spec/adjacent_spec.rb

```diff
@@ -0,0 +1,176 @@
+require File.expand_path('watirspec/spec_helper', File.dirname(__FILE__))
+
+describe "Adjacent Elements" do
```

Comment:
Sure? I mean, I'm assuming everything will end up in the same place after we pull watirspecs into watir. I'm just putting new stuff in `spec/` because it is easier to run. ¯\_(ツ)_/¯